packages/presentation/lib/screen/login/login_bloc.dart

```diff
@@ -85,24 +86,24 @@ class _LoginBloc extends BlocImpl<LoginScreenArguments, LoginData>
     final String email = _emailController.text.trim();
     final String password = _passwordController.text.trim();
     final UserModel user = UserModel(email, password);
-    final LoginValidationError? validationResponse =
-        await _loginValidationUseCase(user);
-    validationModel = ValidationModel(
-      validationResponse?.emailValidation,
-      validationResponse?.passwordValidation,
-    );
-    if (_formKey.currentState?.validate() ?? false) {
-      bool isSuccess = await _checkUserUseCase(user);
-      if (!isSuccess) {
-        validationModel = ValidationModel(
-          ValidationErrorType.invalidValue,
-          ValidationErrorType.invalidValue,
-        );
-        _formKey.currentState?.validate();
-      } else {
+    try {
+      final LoginValidationError? validationResponse =
+          await _loginValidationUseCase(user);
+      validationModel = ValidationModel(
+        validationResponse?.emailValidation,
+        validationResponse?.passwordValidation,
+      );
+      if (_formKey.currentState?.validate() ?? false) {
```

Comment:
still not good - u should call form.validate only in the catch block. it shouldn't be in try